population/client.go

```diff
@@ -405,6 +415,70 @@ func (c *Client) GetAreaTypeParents(ctx context.Context, input GetAreaTypeParent
 	return atp, nil
 }
 
+func (c *Client) GetParentAreaCount(ctx context.Context, input GetParentsAreaCountInput) (int, error) {
+	logData := log.Data{
+		"method":              http.MethodGet,
+		"dataset_id":          input.DatasetID,
+		"area_type_id":        input.AreaTypeID,
+		"parent_area_type_id": input.ParentAreaTypeID,
+		"areas":               input.Areas,
+	}
+
+	urlPath := fmt.Sprintf("population-types/%s/area-types/%s/parents/%s/areas-count",
+		input.DatasetID, input.AreaTypeID, input.ParentAreaTypeID)
```

Comment:
More Go-typical formatting might look something like:

```
urlPath := fmt.Sprintf(
  "population-types/%s/area-types/%s/parents/%s/areas-count",
  input.DatasetID,
  input.AreaTypeID,
  input.ParentAreaTypeID,
)
```
Certainly close the bracket on it's own line

Reply:
This has been addressed.